dsl/src/main/java/tech/pegasys/peeps/node/GoQuorum.java

```diff
@@ -37,33 +38,42 @@
   private static final String IMAGE_NAME = "quorumengineering/quorum:latest";
   private static final String CONTAINER_GENESIS_FILE = "/etc/genesis.json";
   private static final String CONTAINER_NODE_PRIVATE_KEY_FILE = "/etc/keys/node.priv";
+  private static final String CONTAINER_PASSWORD_FILE = "/etc/password";
```

Comment:
think it's a bad idea using this for the password file as this masks the normal unix password file. maybe just put all these files under /etc/goquorum or /etc/web3provider so they can't mess with system files